.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
```

Comment:
This might seem silly but I think adding an emoji to the beginning of the `name` helps people do a good visual scan and not just click blindly.

Like 🐛 or ❗️ or something for bug

Reply:
SGTM